codes/train_ppon.py

```diff
@@ -1,296 +0,0 @@
-import os.path
```

Comment:
Keep file, will modify for an image to image translation based upscale WBC+PPON

Reply:
See comments relating to test_ppon.py